Sources/Charcoal/Filters/Root/Views/BottomButtonCalloutOverlay.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+
+protocol BottomButtonCalloutOverlayDelegate: AnyObject {
+    func bottomButtomCalloutOverlayDidTapInside(_ view: BottomButtonCalloutOverlay)
+}
+
+final class BottomButtonCalloutOverlay: UIView {
+    weak var delegate: BottomButtonCalloutOverlayDelegate?
+
+    // MARK: - Private properties
+
+    private lazy var calloutView: CalloutView = {
+        let view = CalloutView(direction: .down)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    private lazy var bodyView: UIView = {
+        let view = UIView(withAutoLayout: true)
+        view.backgroundColor = UIColor(white: 1, alpha: 0.8)
+        return view
+    }()
+
+    // MARK: - Init
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Setup
+
+    func configure(withText text: String) {
+        calloutView.show(withText: text, duration: 0)
+    }
+
+    private func setup() {
+        backgroundColor = .clear
+
+        addSubview(bodyView)
+        addSubview(calloutView)
+
+        let gestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(handleTap))
+        addGestureRecognizer(gestureRecognizer)
+
+        NSLayoutConstraint.activate([
+            calloutView.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -76),
```

Comment:
I'm not happy with this magic constant that is equal to the height of the `FilterBottomButtonView` height. If there are any tips on how to get the height I'll gladly take them 🙃 One solution I tried out were to get the height from the `intrinsicContentSize` on `FilterBottomButtonView` when making the bottom button visible and pass it through the `configure(withText:)` method, but it felt dirty 🤔

Reply:
I guess you can constrain it to the `topAnchor` of the bottomButton. You can pass an anchor to the configure method f.eks.
`configure(withText text: String, anchor: NSLayoutYAxisAnchor)`